hieradata/role/jenkins-slave.yaml

```diff
@@ -1,12 +1,4 @@
 ---
-psick::base::windows_classes:
```

Comment:
Did you check it without psick::openssh, we need it to psick::openssh::configs_hash:

Reply:
No, I don't have an environment to test, please test it. Also, please put all comments in a single review rather than sending separate comments.